source/Plugins/Language/Swift/FoundationValueTypes.cpp

```diff
@@ -282,7 +284,8 @@ class URLComponentsSyntheticChildrenFrontEnd
     : public SyntheticChildrenFrontEnd {
 public:
   URLComponentsSyntheticChildrenFrontEnd(lldb::ValueObjectSP valobj_sp)
-      : SyntheticChildrenFrontEnd(*valobj_sp), m_synth_up(nullptr)
+      : SyntheticChildrenFrontEnd(*valobj_sp), m_synth_frontend_up(),
+        m_synth_backend_up()
```

Comment:
Not really part of this patch, but: I suppose these should be named `_sp` (smartpointer) instead of `_up`?